controls/accordion/accordion-item.md

```diff
@@ -27,11 +27,11 @@ The indicator is the little triangle that is rotated according to whether the Ac
 
 ## IsExpanded
 
-Change the expanded state of the AccordionItem using the `IsExpanded`(`bool`)property.
+Change the expanded state of the `AccordionItem` using the `IsExpanded`(`bool`)property.
 	
 ## Border Styles
 
-Utilize `BorderColor`, `BorderThickness` and `CornerRadius` properties of RadAccordionItem to change the way the Border around the control looks.
+Use `BorderColor`, `BorderThickness` and `CornerRadius` properties of `RadAccordionItem` to change the way the Border around the control looks.
```

Comment:
Always use the definite article when referring to a specific noun or unique object!
When you describe the "property" as "BorderColor", this makes it specific. It is not any property, it is **the BorderColor property**. 
```suggestion
Use the `BorderColor`, `BorderThickness`, and `CornerRadius` properties of `RadAccordionItem` to change the way the Border around the control looks.
```